.github/workflows/develocity-publish-build-scans.yml

```diff
@@ -19,15 +19,16 @@ jobs:
       checks: write
     steps:
       - name: Setup Build Scan link capture
-        uses: gradle/github-actions/maven-build-scan-setup@v0.2
+        uses: gradle/develocity-actions/maven-setup@v1
+        with:
+          add-pr-comment: false
+          add-job-summary: true
```

Comment:
Is there a reason why you switched this one to `true`?

IIRC, I decided to handle it myself as the ordering wasn't ideal and I adjusted it. Does your job summary provide more value than the one I add below with my action?

Reply:
This is meant to help troubleshoot the potentially missing Build Scan publications that were encountered recently.
We agreed to enable it temporarily at least.